physionet-django/project/templates/project/published_project.html

```diff
@@ -377,6 +375,8 @@ <h5>Access the files</h5>
           </p>
         {% endif %}
 
+        {% if project.allow_file_downloads %}
```

Comment:
Will the ever be `False`? I think this falls under the `has_access` condition:
```
{% if has_access %}
  {% if project.allow_file_downloads %}
  {% endif %}
{% endif %}
```
But if `project.allow_file_downloads` is `False`, then `has_access` will be `False` as well: https://github.com/MIT-LCP/physionet-build/blob/291a60b57cf1847f4b8da96573ed361784c2b882/physionet-django/project/modelcomponents/publishedproject.py#L218-L219

We'll need to add a distinction between having access to the project and being able to download the files.